Corina's first project

Cocolina-90 opened the pull request:
Hi, I would like to hand in my first project. 
Once you have checked my codes, I might go on and fill in the other menu tabs such as "Photo Gallery" etc. 
I have not published the page yet, as I would have to review the content before doing so. There might be some spelling mistakes as I was more focusing on structure and design than the content itself. 

Many thanks for your feedback. 

Best regards,
Corina

Cocolina-90 committed:
added my first project

added the first project including some additional images and html files

Cocolina-90 committed:
Update README.md

HIPPIEKICK commented on README.md:
```diff
@@ -1,12 +1,15 @@
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Since I did not have the intention to right a blog at all, my first problem was to find a suitable subject. Once I had an idea in mind, I made a sketch with pen and paper regarding my design and the content (headings). 
+Based on this sketch, I tried to code. Whenever I got stuck, I googled for an answer and it actually worked. 
```
That's awesome! One of the most important things to learn properly is how to google so it's really good to practice that from the beginning :)

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+    </menu>
+
+<!-- I am sure there is a better way to position the pictures. However, I couldn't figure out yet.
+Please let me know how you would do it. I want them to overlap.-->
```
Sounds like a good idea to use `position: absolute;` if you want them to overlap 👍 Try to keep the styling to the CSS-file though - then it will be much easier to find if you want to change something.

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+
+<!-- I am sure there is a better way to position the pictures. However, I couldn't figure out yet.
+Please let me know how you would do it. I want them to overlap.-->
+    <div class="header">
```
There's actually an HTML element called header 🤩 So instead of `<div>`, you could have `<header>` here

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+    <div class="header">
+        <img src="./Images/beach_1.jpg"   alt="picture of a beach" style="position:absolute; top:30px; left:115px; z-index:30px;">
+        <img src="./Images/sunrise.jpg"   alt="picture of a sunrise" style="position:absolute; top:10px; left:520px; z-index:30px;">
+        <img src="./Images/beach_2.jpg"   alt="picture of a beach" style="position:absolute; top:30px; left:935px; z-index:30px;">
```
Great that you added alt-tags, it's super important for accessibility reasons (or if your images can't be displayed for some reason). If your HTML element has a lot of attributes (or more than one, really), it's a good idea two split them on new lines. That makes it much easier to read than if they are on one long line:
```
<img
    src="./Images/beach_1.jpg"
    alt="picture of a beach">
```

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+        <h2>Unterrichten an der Pazifkküste - ein Erlebnisbericht</h2>
+    </div>
+
+<!-- I wasn't sure if it makes sense to spilt my section into various sections. But as the 
+syntax stays the same for all different parts, I decided to go for just one section.
+Regarding my font: I am well aware its not super easy readable, but I liked it as this blog
+should be some kind of diary. I would probaly not publish it with this font though.-->
```
It's a matter of taste! One could argue that less is more (we should not add HTML unless we need to), but sometimes it's a good idea to do so - to make the HTML more semantic and comprehensible. So one option is to just have everything in `<main>`, and get rid of the `<section>`, but for better structure you could wrap your different content in sections:
```
<main>
    <section>
        <h3></h3>
        <p></p>
    </section>

    <section>
        <h3></h3>
        <p></p>
    </section>

    <section>
        <h3></h3>
        <p></p>
    </section>
</main>
```

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+    <main>
+        <section>
+
+            <h3><span>Worum</span> geht's?</h3>
```
Great usage of `span`!

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+                Ich möchte euch hier mehr über meine Erlebnisse in Costa Rica erzählen. Zum einen
+                in diesem Intro über die wichtigsten W-Fragen, zum anderen aber auch etwas ausführlicher über 
+                das Land Costa Rica, Reisen in Costa Rica und das Leben (bei einer Gastfamilie) in Costa Rica. 
+                Bei Fragen stehe ich euch gerne <a href="mailto:corinafuchs@icloud.com">via Mail</a> zur Verfügung.
```
Nice with a `mailto` 💯

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+            </p>
+
+            <h3><span>Wer</span> bin ich?</h3>
+            <p> 
+                <img src="./Images/IMG_2831.jpg"
+                style="float:left; margin-right:20px; width:150px; height:150px; border-radius:50%;"
+                alt="photo of Corina Fuchs"
+                />
+                Mein Name ist Corina. Ich bin im Februar 1990 im Appenzellerland zur Welt gekommen.
+                Seit über 10 Jahren arbeite ich für eine Schweizer Bank als Kundenberaterin. Ein Job, der
+                mir viel Freude bereitet und dank welchem ich jeden Tag neue Leute kennenlernen darf. 
+                In meiner Freizeit liebe ich es, zu reisen und Neues zu entdecken.
+                Da es in fernen Ländern doppelt so interessant ist, wen man sich mit den Menschen vor Ort austauschen kann, 
+                lerne ich gerne Sprachen. 
+            </p>
```
Hmm, it doesn't really make sense to have an image inside a paragraph of text. Consider wrapping the `img` and `p` in another element e.g. a `div`. And be consistent on whether you choose to close your img-tags or not: `<img>` vs. `<img/>`. And again - try to keep the styling to the CSS-file :)

HIPPIEKICK replied:
Regarding the styling, do you see that the image is a little bit squished together? You could prevent that if you add
`object-fit: cover;`
🙌

HIPPIEKICK commented on code/index.html:
```diff
@@ -1,21 +1,137 @@
+
+    </main>
+
+        <footer>Publiziert am 20.09.2020 von Corina Fuchs</footer>
```
Nice with a footer 👍

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,124 @@
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```
Yey, flexbox 💪

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,124 @@
+  color:rgba(243, 234, 204, 0.897);
+}
+
+@keyframes scale {
+  10% {transform: scale(2);} 
+  100% {transform: scale(1);}
+}
```
Cool with an animation! The scaling makes the image cover the other elements so maybe you want to scale it a little bit less, maybe 1.2?
```
@keyframes scale {
  10% {transform: scale(1.2);} 
  100% {transform: scale(1);}
}
```

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,124 @@
+  text-align: center;
+  width: auto;
+  padding: 5px;
+  margin-left:100px;
+  margin-right: 100px;
```
You could write this in one line like this, where the first value is top/bottom and the second one is left/right:
`margin: 0 100px;`

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,124 @@
+
+.title h1{
+  font-size: 75px;
+  margin-bottom: 10px;
+  margin-top: 5px;
```
You could even give it four different values:
`margin: top, right, bottom, left;`

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,124 @@
+
+h3 {
+  font-size: 40px;
+  margin-bottom: 0px;
```
When the value is zero, you don't have to specify the unit:
`margin-bottom: 0;`

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,124 @@
+p {
+  font-size: 30px;
+  text-align: left;
+  padding: 5px 10px 5px 10px;
```
And here, you could shorten it to two values since top/bottom are the same and left/right are the same

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,124 @@
+  padding-left: 16px; 
+}
+
+li:before {
```
Nice that you're using the `:before` selector, it's super useful!

HIPPIEKICK reviewed (commented):
Wow, thank you for sharing your site, Corina! You've done a great job both with the HTML and the CSS. Your HTML is easy to follow since you have indented it properly, and you've tried a lot of different things out in CSS (like animation, flexbox and different selectors) which is awesome! Great to read your comments as well!

Apart from the things I've already commented on, I want to share a link with you about default values. It's really useful for checking the default values of the different HTML elements so we only specify if we want something to be different from the default: https://www.w3schools.com/cssref/css_default_values.asp

With that said, great job and keep up the good work! ⭐ 

/Matilda
Technigo

Cocolina-90 committed:
I updated the HTML and CSS 

i slightly changed the structure of the HTML so it would be easier to work on the other sites of the page (e.g. I added more classes so that I could easily copy it to other parts, using the same CSS). 
I also included a header and changed the design a bit.

Cocolina-90 commented:
Dear Matilda

Many thanks for your feedback! 

I actually submitted the pull request a bit too early … I had a bit more time than expected and I therefore used the last days to adjust a few things. 
I realised my blog wasn’t responsive at all. So i tried to change that. I just uploaded the new html and css files. 

I will go through your feedback in details the next days and try to implement your tips. 

I had one problem that I couldn’t solve: 
I tried to set a background color for my header. but the color wasn’t applied to all of it and it looked weird specially around the pictures. I couldn’t figure out what the problem was. Since i didn’t need the color for design purpose, i left it how it was but I kept wondering why it wasn’t working the way i thought it should :) 

wishing you a nice evening and I am looking forward to the rest of the course! 

Best regards,
Corina 



> On 23 Sep 2020, at 19:21, Matilda Arvidsson <notifications@github.com> wrote:
> 
> 
> @HIPPIEKICK commented on this pull request.
> 
> Wow, thank you for sharing your site, Corina! You've done a great job both with the HTML and the CSS. Your HTML is easy to follow since you have indented it properly, and you've tried a lot of different things out in CSS (like animation, flexbox and different selectors) which is awesome! Great to read your comments as well!
> 
> Apart from the things I've already commented on, I want to share a link with you about default values. It's really useful for checking the default values of the different HTML elements so we only specify if we want something to be different from the default: https://www.w3schools.com/cssref/css_default_values.asp <https://www.w3schools.com/cssref/css_default_values.asp>
> With that said, great job and keep up the good work! ⭐
> 
> In README.md <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493323852>:
> 
> >  
>  ## The problem
>  
> -Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
> +Since I did not have the intention to right a blog at all, my first problem was to find a suitable subject. Once I had an idea in mind, I made a sketch with pen and paper regarding my design and the content (headings). 
> +Based on this sketch, I tried to code. Whenever I got stuck, I googled for an answer and it actually worked. 
> That's awesome! One of the most important things to learn properly is how to google so it's really good to practice that from the beginning :)
> 
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493327043>:
> 
> > +<!-- I am sure there is a better way to position the pictures. However, I couldn't figure out yet.
> +Please let me know how you would do it. I want them to overlap.-->
> Sounds like a good idea to use position: absolute; if you want them to overlap 👍 Try to keep the styling to the CSS-file though - then it will be much easier to find if you want to change something.
> 
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493328618>:
> 
> > +
> +<body>
> +
> +<!-- I have not filled in the other menu blocks yet. I thought it makes sense that you first
> +check my codes and if it is okay, I will go on.-->
> +    <menu>
> +        <a href="./">Home</a>
> +        <a href="./Reisen in Costa Rica.html">Reisen in Costa Rica</a>
> +        <a href="./photo_gallary.html">Photo Gallery</a>
> +        <a href="./Leben bei einer Gastfamilie.html">Leben bei einer Gastfamilie</a>
> +        <a href="./über Costa Rica.html">über Costa Rica</a>
> +    </menu>
> +
> +<!-- I am sure there is a better way to position the pictures. However, I couldn't figure out yet.
> +Please let me know how you would do it. I want them to overlap.-->
> +    <div class="header">
> There's actually an HTML element called header 🤩 So instead of <div>, you could have <header> here
> 
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493330439>:
> 
> > +        <img src="./Images/beach_1.jpg"   alt="picture of a beach" style="position:absolute; top:30px; left:115px; z-index:30px;">
> +        <img src="./Images/sunrise.jpg"   alt="picture of a sunrise" style="position:absolute; top:10px; left:520px; z-index:30px;">
> +        <img src="./Images/beach_2.jpg"   alt="picture of a beach" style="position:absolute; top:30px; left:935px; z-index:30px;">
> Great that you added alt-tags, it's super important for accessibility reasons (or if your images can't be displayed for some reason). If your HTML element has a lot of attributes (or more than one, really), it's a good idea two split them on new lines. That makes it much easier to read than if they are on one long line:
> 
> <img
>     src="./Images/beach_1.jpg"
>     alt="picture of a beach">
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493336730>:
> 
> > +<!-- I wasn't sure if it makes sense to spilt my section into various sections. But as the 
> +syntax stays the same for all different parts, I decided to go for just one section.
> +Regarding my font: I am well aware its not super easy readable, but I liked it as this blog
> +should be some kind of diary. I would probaly not publish it with this font though.-->
> It's a matter of taste! One could argue that less is more (we should not add HTML unless we need to), but sometimes it's a good idea to do so - to make the HTML more semantic and comprehensible. So one option is to just have everything in <main>, and get rid of the <section>, but for better structure you could wrap your different content in sections:
> 
> <main>
>     <section>
>         <h3></h3>
>         <p></p>
>     </section>
> 
>     <section>
>         <h3></h3>
>         <p></p>
>     </section>
> 
>     <section>
>         <h3></h3>
>         <p></p>
>     </section>
> </main>
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493337446>:
> 
> > +        <img src="./Images/beach_2.jpg"   alt="picture of a beach" style="position:absolute; top:30px; left:935px; z-index:30px;">
> +    </div>
> +
> +    <div class="title">
> +        <h1>Volunteering in Costa Rica</h1>
> +        <h2>Unterrichten an der Pazifkküste - ein Erlebnisbericht</h2>
> +    </div>
> +
> +<!-- I wasn't sure if it makes sense to spilt my section into various sections. But as the 
> +syntax stays the same for all different parts, I decided to go for just one section.
> +Regarding my font: I am well aware its not super easy readable, but I liked it as this blog
> +should be some kind of diary. I would probaly not publish it with this font though.-->
> +    <main>
> +        <section>
> +
> +            <h3><span>Worum</span> geht's?</h3>
> Great usage of span!
> 
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493337650>:
> 
> > +        <h2>Unterrichten an der Pazifkküste - ein Erlebnisbericht</h2>
> +    </div>
> +
> +<!-- I wasn't sure if it makes sense to spilt my section into various sections. But as the 
> +syntax stays the same for all different parts, I decided to go for just one section.
> +Regarding my font: I am well aware its not super easy readable, but I liked it as this blog
> +should be some kind of diary. I would probaly not publish it with this font though.-->
> +    <main>
> +        <section>
> +
> +            <h3><span>Worum</span> geht's?</h3>
> +            <p>Hallo und herzlich willkommen auf meinem Blog über Volunteering in Costa Rica.
> +                Ich möchte euch hier mehr über meine Erlebnisse in Costa Rica erzählen. Zum einen
> +                in diesem Intro über die wichtigsten W-Fragen, zum anderen aber auch etwas ausführlicher über 
> +                das Land Costa Rica, Reisen in Costa Rica und das Leben (bei einer Gastfamilie) in Costa Rica. 
> +                Bei Fragen stehe ich euch gerne <a href="mailto:corinafuchs@icloud.com">via Mail</a> zur Verfügung.
> Nice with a mailto 💯
> 
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493341682>:
> 
> > +            <p> 
> +                <img src="./Images/IMG_2831.jpg"
> +                style="float:left; margin-right:20px; width:150px; height:150px; border-radius:50%;"
> +                alt="photo of Corina Fuchs"
> +                />
> +                Mein Name ist Corina. Ich bin im Februar 1990 im Appenzellerland zur Welt gekommen.
> +                Seit über 10 Jahren arbeite ich für eine Schweizer Bank als Kundenberaterin. Ein Job, der
> +                mir viel Freude bereitet und dank welchem ich jeden Tag neue Leute kennenlernen darf. 
> +                In meiner Freizeit liebe ich es, zu reisen und Neues zu entdecken.
> +                Da es in fernen Ländern doppelt so interessant ist, wen man sich mit den Menschen vor Ort austauschen kann, 
> +                lerne ich gerne Sprachen. 
> +            </p>
> Hmm, it doesn't really make sense to have an image inside a paragraph of text. Consider wrapping the img and p in another element e.g. a div. And be consistent on whether you choose to close your img-tags or not: <img> vs. <img/>. And again - try to keep the styling to the CSS-file :)
> 
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493737466>:
> 
> > +                braucht wenig Materielles, um glücklich zu sein. Mehr details hierzu findet ihr unter dem
> +                Abschnitt "Leben bei einer Gastfamilie". 
> +            </p>
> +
> +            <h3><span>Wem</span> würde ich es weitermpfehlen?</h3>
> +            <p>Jedem, der gerne in eine anderer Kultur eintauchen möchte. Dazu gehört aber auch, dass man 
> +                Gewisse Gewohnheiten ablegt und sich an neue Umstände anpassen kann. Auf dem Lande geht es
> +                sehr ruhig zu und her und somit ist man defintiv falsch, wenn man die grossen Parties sucht.
> +                Dafür kennt man nach kurzer Zeit das ganze Dorf und hat die einmalige Gelegenheit, neue Perspektiven
> +                kennenzulernen und davon zu lernen. 
> +            </p>
> +        </section>
> +
> +    </main>
> +
> +        <footer>Publiziert am 20.09.2020 von Corina Fuchs</footer>
> Nice with a footer 👍
> 
> In code/style.css <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493738801>:
> 
> > +menu {
> +  display: flex;
> +  align-items: center;
> +  justify-content: center;
> +}
> Yey, flexbox 💪
> 
> In code/style.css <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493749879>:
> 
> > +@keyframes scale {
> +  10% {transform: scale(2);} 
> +  100% {transform: scale(1);}
> +}
> Cool with an animation! The scaling makes the image cover the other elements so maybe you want to scale it a little bit less, maybe 1.2?
> 
> @keyframes scale {
>   10% {transform: scale(1.2);} 
>   100% {transform: scale(1);}
> }
> In code/style.css <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493751405>:
> 
> > +  margin-left:100px;
> +  margin-right: 100px;
> You could write this in one line like this, where the first value is top/bottom and the second one is left/right:
> margin: 0 100px;
> 
> In code/style.css <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493752317>:
> 
> > +  margin-bottom: 10px;
> +  margin-top: 5px;
> You could even give it four different values:
> margin: top, right, bottom, left;
> 
> In code/style.css <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493752646>:
> 
> > +  font-size: 55px;
> +  margin-bottom: 25px;
> +  margin-top: 5px;
> +  color: blue;
> +}
> +
> +section {
> +  text-align: center;
> +  width: auto;
> +  margin-left: 100px;
> +  margin-right: 100px;
> +}
> +
> +h3 {
> +  font-size: 40px;
> +  margin-bottom: 0px;
> When the value is zero, you don't have to specify the unit:
> margin-bottom: 0;
> 
> In code/style.css <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493753989>:
> 
> > +
> +h3 {
> +  font-size: 40px;
> +  margin-bottom: 0px;
> +  margin-top: 40px;
> +}
> +
> +h3>span {
> +  color: red;
> +  font-size: 80px;
> +}
> +
> +p {
> +  font-size: 30px;
> +  text-align: left;
> +  padding: 5px 10px 5px 10px;
> And here, you could shorten it to two values since top/bottom are the same and left/right are the same
> 
> In code/style.css <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493754703>:
> 
> > +  width: auto;
> +}
> +
> +ul {
> +  list-style: none; /* Remove list bullets */
> +  padding: 0;
> +  margin: 0;
> +  text-align: left;
> +  font-size: 30px;
> +}
> +
> +li { 
> +  padding-left: 16px; 
> +}
> +
> +li:before {
> Nice that you're using the :before selector, it's super useful!
> 
> In code/index.html <https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493756358>:
> 
> > +            <p> 
> +                <img src="./Images/IMG_2831.jpg"
> +                style="float:left; margin-right:20px; width:150px; height:150px; border-radius:50%;"
> +                alt="photo of Corina Fuchs"
> +                />
> +                Mein Name ist Corina. Ich bin im Februar 1990 im Appenzellerland zur Welt gekommen.
> +                Seit über 10 Jahren arbeite ich für eine Schweizer Bank als Kundenberaterin. Ein Job, der
> +                mir viel Freude bereitet und dank welchem ich jeden Tag neue Leute kennenlernen darf. 
> +                In meiner Freizeit liebe ich es, zu reisen und Neues zu entdecken.
> +                Da es in fernen Ländern doppelt so interessant ist, wen man sich mit den Menschen vor Ort austauschen kann, 
> +                lerne ich gerne Sprachen. 
> +            </p>
> Regarding the styling, do you see that the image is a little bit squished together? You could prevent that if you add
> object-fit: cover;
> 🙌
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/Technigo/project-p1-first-webpage/pull/8#pullrequestreview-494435096>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQ5BREJBQSYVOUEOAQOP3MDSHIVBDANCNFSM4RSH3VDQ>.
>

testmatilda commented:
> Dear Matilda Many thanks for your feedback! I actually submitted the pull request a bit too early … I had a bit more time than expected and I therefore used the last days to adjust a few things. I realised my blog wasn’t responsive at all. So i tried to change that. I just uploaded the new html and css files. I will go through your feedback in details the next days and try to implement your tips. I had one problem that I couldn’t solve: I tried to set a background color for my header. but the color wasn’t applied to all of it and it looked weird specially around the pictures. I couldn’t figure out what the problem was. Since i didn’t need the color for design purpose, i left it how it was but I kept wondering why it wasn’t working the way i thought it should :) wishing you a nice evening and I am looking forward to the rest of the course! Best regards, Corina
> […](#)
> On 23 Sep 2020, at 19:21, Matilda Arvidsson ***@***.***> wrote: @HIPPIEKICK commented on this pull request. Wow, thank you for sharing your site, Corina! You've done a great job both with the HTML and the CSS. Your HTML is easy to follow since you have indented it properly, and you've tried a lot of different things out in CSS (like animation, flexbox and different selectors) which is awesome! Great to read your comments as well! Apart from the things I've already commented on, I want to share a link with you about default values. It's really useful for checking the default values of the different HTML elements so we only specify if we want something to be different from the default: https://www.w3schools.com/cssref/css_default_values.asp <https://www.w3schools.com/cssref/css_default_values.asp> With that said, great job and keep up the good work! ⭐ In README.md <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493323852)>: > ## The problem -Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next? +Since I did not have the intention to right a blog at all, my first problem was to find a suitable subject. Once I had an idea in mind, I made a sketch with pen and paper regarding my design and the content (headings). +Based on this sketch, I tried to code. Whenever I got stuck, I googled for an answer and it actually worked. That's awesome! One of the most important things to learn properly is how to google so it's really good to practice that from the beginning :) In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493327043)>: > +<!-- I am sure there is a better way to position the pictures. However, I couldn't figure out yet. +Please let me know how you would do it. I want them to overlap.--> Sounds like a good idea to use position: absolute; if you want them to overlap 👍 Try to keep the styling to the CSS-file though - then it will be much easier to find if you want to change something. In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493328618)>: > + +<body> + +<!-- I have not filled in the other menu blocks yet. I thought it makes sense that you first +check my codes and if it is okay, I will go on.--> + <menu> + <a href="./">Home</a> + <a href="./Reisen in Costa Rica.html">Reisen in Costa Rica</a> + <a href="./photo_gallary.html">Photo Gallery</a> + <a href="./Leben bei einer Gastfamilie.html">Leben bei einer Gastfamilie</a> + <a href="./über Costa Rica.html">über Costa Rica</a> + </menu> + +<!-- I am sure there is a better way to position the pictures. However, I couldn't figure out yet. +Please let me know how you would do it. I want them to overlap.--> + <div class="header"> There's actually an HTML element called header 🤩 So instead of <div>, you could have <header> here In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493330439)>: > + <img src="./Images/beach_1.jpg" alt="picture of a beach" style="position:absolute; top:30px; left:115px; z-index:30px;"> + <img src="./Images/sunrise.jpg" alt="picture of a sunrise" style="position:absolute; top:10px; left:520px; z-index:30px;"> + <img src="./Images/beach_2.jpg" alt="picture of a beach" style="position:absolute; top:30px; left:935px; z-index:30px;"> Great that you added alt-tags, it's super important for accessibility reasons (or if your images can't be displayed for some reason). If your HTML element has a lot of attributes (or more than one, really), it's a good idea two split them on new lines. That makes it much easier to read than if they are on one long line: <img src="./Images/beach_1.jpg" alt="picture of a beach"> In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493336730)>: > +<!-- I wasn't sure if it makes sense to spilt my section into various sections. But as the +syntax stays the same for all different parts, I decided to go for just one section. +Regarding my font: I am well aware its not super easy readable, but I liked it as this blog +should be some kind of diary. I would probaly not publish it with this font though.--> It's a matter of taste! One could argue that less is more (we should not add HTML unless we need to), but sometimes it's a good idea to do so - to make the HTML more semantic and comprehensible. So one option is to just have everything in <main>, and get rid of the <section>, but for better structure you could wrap your different content in sections: <main> <section> <h3></h3> <p></p> </section> <section> <h3></h3> <p></p> </section> <section> <h3></h3> <p></p> </section> </main> In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493337446)>: > + <img src="./Images/beach_2.jpg" alt="picture of a beach" style="position:absolute; top:30px; left:935px; z-index:30px;"> + </div> + + <div class="title"> + <h1>Volunteering in Costa Rica</h1> + <h2>Unterrichten an der Pazifkküste - ein Erlebnisbericht</h2> + </div> + +<!-- I wasn't sure if it makes sense to spilt my section into various sections. But as the +syntax stays the same for all different parts, I decided to go for just one section. +Regarding my font: I am well aware its not super easy readable, but I liked it as this blog +should be some kind of diary. I would probaly not publish it with this font though.--> + <main> + <section> + + <h3><span>Worum</span> geht's?</h3> Great usage of span! In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493337650)>: > + <h2>Unterrichten an der Pazifkküste - ein Erlebnisbericht</h2> + </div> + +<!-- I wasn't sure if it makes sense to spilt my section into various sections. But as the +syntax stays the same for all different parts, I decided to go for just one section. +Regarding my font: I am well aware its not super easy readable, but I liked it as this blog +should be some kind of diary. I would probaly not publish it with this font though.--> + <main> + <section> + + <h3><span>Worum</span> geht's?</h3> + <p>Hallo und herzlich willkommen auf meinem Blog über Volunteering in Costa Rica. + Ich möchte euch hier mehr über meine Erlebnisse in Costa Rica erzählen. Zum einen + in diesem Intro über die wichtigsten W-Fragen, zum anderen aber auch etwas ausführlicher über + das Land Costa Rica, Reisen in Costa Rica und das Leben (bei einer Gastfamilie) in Costa Rica. + Bei Fragen stehe ich euch gerne <a ***@***.***">via Mail</a> zur Verfügung. Nice with a mailto 💯 In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493341682)>: > + <p> + <img src="./Images/IMG_2831.jpg" + style="float:left; margin-right:20px; width:150px; height:150px; border-radius:50%;" + alt="photo of Corina Fuchs" + /> + Mein Name ist Corina. Ich bin im Februar 1990 im Appenzellerland zur Welt gekommen. + Seit über 10 Jahren arbeite ich für eine Schweizer Bank als Kundenberaterin. Ein Job, der + mir viel Freude bereitet und dank welchem ich jeden Tag neue Leute kennenlernen darf. + In meiner Freizeit liebe ich es, zu reisen und Neues zu entdecken. + Da es in fernen Ländern doppelt so interessant ist, wen man sich mit den Menschen vor Ort austauschen kann, + lerne ich gerne Sprachen. + </p> Hmm, it doesn't really make sense to have an image inside a paragraph of text. Consider wrapping the img and p in another element e.g. a div. And be consistent on whether you choose to close your img-tags or not: <img> vs. <img/>. And again - try to keep the styling to the CSS-file :) In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493737466)>: > + braucht wenig Materielles, um glücklich zu sein. Mehr details hierzu findet ihr unter dem + Abschnitt "Leben bei einer Gastfamilie". + </p> + + <h3><span>Wem</span> würde ich es weitermpfehlen?</h3> + <p>Jedem, der gerne in eine anderer Kultur eintauchen möchte. Dazu gehört aber auch, dass man + Gewisse Gewohnheiten ablegt und sich an neue Umstände anpassen kann. Auf dem Lande geht es + sehr ruhig zu und her und somit ist man defintiv falsch, wenn man die grossen Parties sucht. + Dafür kennt man nach kurzer Zeit das ganze Dorf und hat die einmalige Gelegenheit, neue Perspektiven + kennenzulernen und davon zu lernen. + </p> + </section> + + </main> + + <footer>Publiziert am 20.09.2020 von Corina Fuchs</footer> Nice with a footer 👍 In code/style.css <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493738801)>: > +menu { + display: flex; + align-items: center; + justify-content: center; +} Yey, flexbox 💪 In code/style.css <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493749879)>: > ***@***.*** scale { + 10% {transform: scale(2);} + 100% {transform: scale(1);} +} Cool with an animation! The scaling makes the image cover the other elements so maybe you want to scale it a little bit less, maybe 1.2? @Keyframes scale { 10% {transform: scale(1.2);} 100% {transform: scale(1);} } In code/style.css <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493751405)>: > + margin-left:100px; + margin-right: 100px; You could write this in one line like this, where the first value is top/bottom and the second one is left/right: margin: 0 100px; In code/style.css <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493752317)>: > + margin-bottom: 10px; + margin-top: 5px; You could even give it four different values: margin: top, right, bottom, left; In code/style.css <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493752646)>: > + font-size: 55px; + margin-bottom: 25px; + margin-top: 5px; + color: blue; +} + +section { + text-align: center; + width: auto; + margin-left: 100px; + margin-right: 100px; +} + +h3 { + font-size: 40px; + margin-bottom: 0px; When the value is zero, you don't have to specify the unit: margin-bottom: 0; In code/style.css <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493753989)>: > + +h3 { + font-size: 40px; + margin-bottom: 0px; + margin-top: 40px; +} + +h3>span { + color: red; + font-size: 80px; +} + +p { + font-size: 30px; + text-align: left; + padding: 5px 10px 5px 10px; And here, you could shorten it to two values since top/bottom are the same and left/right are the same In code/style.css <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493754703)>: > + width: auto; +} + +ul { + list-style: none; /* Remove list bullets */ + padding: 0; + margin: 0; + text-align: left; + font-size: 30px; +} + +li { + padding-left: 16px; +} + +li:before { Nice that you're using the :before selector, it's super useful! In code/index.html <[#8 (comment)](https://github.com/Technigo/project-p1-first-webpage/pull/8#discussion_r493756358)>: > + <p> + <img src="./Images/IMG_2831.jpg" + style="float:left; margin-right:20px; width:150px; height:150px; border-radius:50%;" + alt="photo of Corina Fuchs" + /> + Mein Name ist Corina. Ich bin im Februar 1990 im Appenzellerland zur Welt gekommen. + Seit über 10 Jahren arbeite ich für eine Schweizer Bank als Kundenberaterin. Ein Job, der + mir viel Freude bereitet und dank welchem ich jeden Tag neue Leute kennenlernen darf. + In meiner Freizeit liebe ich es, zu reisen und Neues zu entdecken. + Da es in fernen Ländern doppelt so interessant ist, wen man sich mit den Menschen vor Ort austauschen kann, + lerne ich gerne Sprachen. + </p> Regarding the styling, do you see that the image is a little bit squished together? You could prevent that if you add object-fit: cover; 🙌 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#8 (review)](https://github.com/Technigo/project-p1-first-webpage/pull/8#pullrequestreview-494435096)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQ5BREJBQSYVOUEOAQOP3MDSHIVBDANCNFSM4RSH3VDQ>.

Great to hear Corina!
You can always go back to projects to refine them as you figure new things out, that's super good! Regarding the background, if the header, there are two main issues:
- Your images take up more width than the body element has place for. The header on the other hand, only takes up the width of the body - so if you make your images smaller (so they fit in the header) you will see some improvement
- The header and the body has some default margin. If you get rid of those margins you will see that the header will stretch all the way to the sides.

Hope this solves your issues!

All the best ⭐ 
/Matilda

HIPPIEKICK commented:
> Dear Matilda Many thanks for your feedback! I actually submitted the pull request a bit too early … I had a bit more time than expected and I therefore used the last days to adjust a few things. I realised my blog wasn’t responsive at all. So i tried to change that. I just uploaded the new html and css files. I will go through your feedback in details the next days and try to implement your tips. I had one problem that I couldn’t solve: I tried to set a background color for my header. but the color wasn’t applied to all of it and it looked weird specially around the pictures. I couldn’t figure out what the problem was. Since i didn’t need the color for design purpose, i left it how it was but I kept wondering why it wasn’t working the way i thought it should :) wishing you a nice evening and I am looking forward to the rest of the course! Best regards, Corina
> […](#)

Great to hear Corina!
You can always go back to projects to refine them as you figure new things out, that's super good! Regarding the background, if the header, there are two main issues:

- Your images take up more width than the body element has place for. The header on the other hand, only takes up the width of the body - so if you make your images smaller (so they fit in the header) you will see some improvement
- The header and the body has some default margin. If you get rid of those margins you will see that the header will stretch all the way to the sides.

Hope this solves your issues!

All the best ⭐
/Matilda
